evil-dataset/README.md

```diff
@@ -0,0 +1,78 @@
+# Evil Dataset
+
+This directory is intended to house the "evil dataset"--a collection of data & associated queries that test edge cases and behavior changes between versions.
```

Comment:
[Blocker] Just trying to understand your intent.  Are you envisioning this housing all of our "expectations"?